src/routes/Routes.js

```diff
@@ -15,7 +15,8 @@ import {
   MobileView,
 } from "react-device-detect";
 
-export const baseUrl = "/flower/client/";
+export const baseUrl = "/";
+// export const baseUrl = "/flower/client/";
```

Comment:
Base Url is already removed from the BrowserRouter tag.

Reply:
Ok mein inko undo kr leta hu

Reply:
ok